docs/_data/_routes.yml

```diff
@@ -142,3 +142,6 @@ platforms:
         title: Workspace & Domain Settings
         icon: /assets/images/shield.svg
         description: Discover how to set up and manage your workspace, define user permissions, and implement domain-level rules.
+
+      - href: Security
+        title: Security
```

Comment:
No icon or description?

Reply:
@pecanoro I'm not sure what else I'm supposed to include; I followed this [SO](https://stackoverflowteams.com/c/expensify/questions/15135/15136#15136) for instructions. Please see step 10!

Reply:
The other ones seems to have an icon and description, that's why I was wondering. I am not sure if all are supposed to include it or only some:

```

      - href: workspace-and-domain-settings
        title: Workspace & Domain Settings
        icon: /assets/images/shield.svg
        description: Discover how to set up and manage your workspace, define user permissions, and implement domain-level rules.
```